include/glow/Graph/Node.h

```diff
@@ -231,7 +231,7 @@ class Node : public Named,
   ~Node() { releaseUsers(); }
 
   /// \returns the n'th result type of the node.
-  TypeRef getType(unsigned idx = -1) const;
+  TypeRef getType(unsigned idx) const;
```

Comment:
Node::getType has a special case for idx == -1.
This is not used anymore, right?
In other words, we should remove that code (I don't think this PR does that).